### ISSUE TITLE
dont works on boolean or integers

### ISSUE THREAD

**Issue by stephjj** [rsenden/node-red-contrib-property-mapping#3]
on map-map, if the input is a boolean, it doesnt works for the false input.
It's works for the true value

payload: true => payload: "ON"
payload: false => payload: "Unknown: false"

The mapping is
true = ON
false = OFF

**Comment by woozar** [comment]
It is the same for integers, in the case of "0".
Just a wild guess, but it might be, that the input is being validated like "if (msg.payload[path])" with is false if the value is "false" or "0".

**Comment by woozar** [comment]
I believe map.js line 74 "if ( !keyValue ) { return null; }" causes this behavior.

**Comment by rsenden** [comment]
I agree that the code could be improved to properly handle values that evaluate to false. Apart from map.js line 74, I think there may be other places where undefined/null/false/0 handling could be improved. 

Unfortunately at the moment I don't have time to investigate which code requires improvements to properly handle such edge cases, implement the necessary changes, and set up test scenarios for testing these edge cases.

As such, for now you will need to use some work-around. For example, a standard 'change' node could be used for the boolean mapping. Alternatively, you could try converting the payload into a string before passing the message to one of the node-red-contrib-map nodes.

Of course if you are willing to implement and test the necessary fixes to handle these edge cases, I'm more than willing to accept a pull request.
